src/types.ts

```diff
@@ -0,0 +1,7 @@
+export type User = {
+  id: string;
+  username: string;
+  email: string;
+  rolle: "user" | "admin" | "tierheim";
```

Comment:
```suggestion
  role: "user" | "admin" | "tierheim";
```
Bitte überall anpassen.